cmd/thor/node/packer_loop.go

```diff
@@ -190,6 +191,10 @@ func (n *Node) pack(flow *packer.Flow) (err error) {
 		n.processFork(newBlock, oldBest.Header.ID())
 		commitElapsed := mclock.Now() - startTime - execElapsed
 
+		if newBlock.Header().Number() == n.forkConfig.GALACTICA {
+			fmt.Println(fork.GalacticaAsciiArt)
```

Comment:
should the logger be used instead of fmt ?

Reply:
I've tried with it but cannot get it formatted correctly 😅

Reply:
Logger will have the prefix `INFO [04-09|17:15:49.145]`

Reply:
Gotcha 👍